
const errorhanlder=(error,req,res,next)=>
return res.status(400).send(error.message); 
};
module.exports=errorhandler;








try{
const user=getUser(); 
if(!user){ 
throw new Error("user not found");
}
}catch(error)
return next(error)
}
return res.status(200).json({success:true});
});
